post-processing-code/oncho/testRun.py

```diff
@@ -0,0 +1,38 @@
+import sys
+sys.path.append('../general-functions/')
+from aggregation import aggregatePostProcessedFiles, iuLevelAggregate, countryLevelAggregate, africaLevelAggregate
+from general_post_processing import processSingleFile
+import os
+import pandas as pd
+
+oncho_dir = "../../input-data/oncho/"
+scenarios = os.listdir(oncho_dir)
+scenario_index = 0
+print(f'Total scenarios to process: {len(scenarios)}')
+for oncho_scenario in scenarios:
+    scenario_index += 1
+    for country in os.listdir(os.path.join(oncho_dir, oncho_scenario)):
+        if country.endswith("DS_Store"):
+                continue
+        for iu in os.listdir(os.path.join(oncho_dir, oncho_scenario, country)):
+            if iu.endswith("DS_Store"):
+                continue
+            for output_file in os.listdir(os.path.join(oncho_dir, oncho_scenario, country, iu)):
+                if (not(output_file.endswith("raw_all_age_data.csv"))): 
+                    continue     
+                processSingleFile(
+                    df=pd.read_csv(os.path.join(oncho_dir, oncho_scenario, country, iu, output_file)),
+                    scenario=oncho_scenario,
+                    iuName=iu,
+                    prevalence_marker_name="prevalence",
+                    post_processing_start_time=1970,
+                    measure_summary_map={"all": "default"}
```

Comment:
With oncho, there are just a lot of measures that can be summarized, so this was more of a QoL parameter. I am more than happy to change it so that all measures need to be specified (we can have the list of measures as a constant in another file, so we can use list comprehension to quickly/easily make this dict however we want)